java/test/jmri/jmrix/jinput/TreeModelTest.java

```diff
@@ -18,9 +14,10 @@
 public class TreeModelTest {
 
     @Rule
-    public Timeout globalTimeout = Timeout.seconds(10); // 10 second timeout for methods in this test class.
+    public Timeout globalTimeout = Timeout.seconds(20); // timeout (seconds) for all test methods in this test class.
 
     @Test
+    @Ignore("fails in CI if now hardware present, and exception handled internal to library in useless way")
```

Comment:
Not that this fails not because no (not "now") hardware is present, but because JMRI ships a library called jhidrawplugin that includes DLLs only for 32-bit versions of Windows XP and older, and configures all test environments to use it. A better way to silence these exceptions would be to remove the Java System Property `jinput.plugins` from the tests, since by setting that property, we setup the tests to fail. That plugin appears to specifically support the [RailDriver control stand](https://raildriver.com/products/raildriver.php) with a python script, but #4713 makes it possible to use that control stand without relying on scripts or no-longer maintained DLLs.

Reply:
Removing `jinput.plugins` from the AppVeyor test run via `build.xml` seems to work, thanks.

What's the right way to do that for the Travis runs?